src/components/LoginBtn.tsx

```diff
@@ -0,0 +1,25 @@
+import { useSession, signIn, signOut } from 'next-auth/react';
+import { useTranslation } from 'next-i18next';
+
+export default function LoginBtn() {
+  const { data: session } = useSession();
+  const { t } = useTranslation("navbar");
+
+  if (session) {
+    return (
+      <>
+        <button className="block py-2 text-sm justify-between" onClick={() => signOut()}>
+         {t("Sign out")}
+        </button>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <button className="block py-2 text-smjustify-between" onClick={() => signIn()}>
+        {t("Sign in")}
+        </button>
+      </>
+    );
+  }
```

Comment:
❗ 
just a suggestion, you don't need the `else` in the `if else` because you are using `return`

```suggestion
  if (session) return (
      <>
        <button className="block py-2 text-sm justify-between" onClick={() => signOut()}>
         {t("Sign out")}
        </button>
      </>
    );
    
    return (
      <>
        <button className="block py-2 text-smjustify-between" onClick={() => signIn()}>
        {t("Sign in")}
        </button>
      </>
    );
```

Reply:
I havnt push the changes becuase i want to delete the commit whee I used this button because it is moe elated to auth